### PR TITLE
coala org change

### PR DESCRIPTION
**Pull request by adtac**

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -61,4 +61,4 @@ and again cluttering our history. If a reviewer has suggestions, the commit
 shall be amended so the history is not cluttered by “fixup commits”.
 
 For further information/queries feel free to discuss with us on
-`gitter <https://gitter.im/coala-analyzer/coala>`__.
+`gitter <https://gitter.im/coala/coala>`__.
```
Comment (sils1297) [inline CONTRIBUTING.rst]: The channels aren't migrated yet, also use coala.io/chat

Comment (adtac) [inline CONTRIBUTING.rst]: Fixed :+1:

File: README.rst [hunk 2]
```diff
@@ -232,20 +232,20 @@ License
...
    :target: http://coala.rtfd.org/
-.. |AGPL| image:: https://img.shields.io/github/license/coala-analyzer/coala.svg
+.. |AGPL| image:: https://img.shields.io/github/license/coala/coala.svg
    :target: https://www.gnu.org/licenses/agpl-3.0.html
 .. |Gitmate| image:: https://img.shields.io/badge/Gitmate-0%20issues-brightgreen.svg
    :target: http://gitmate.com/
-.. |gitter| image:: https://badges.gitter.im/coala-analyzer/coala.svg
-    :target: https://gitter.im/coala-analyzer/coala
+.. |gitter| image:: https://badges.gitter.im/coala/coala.svg
+    :target: https://gitter.im/coala/coala
```
Comment (Adrianzatreanu) [inline README.rst]: coala.io/chat

**Comment by Adrianzatreanu** [comment]
unack 6a115953e01e451e79f07eaf90c7b032f05e1276

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -10,7 +10,7 @@ design enhancements issues as well as pull Requests to make *coala* more
 convenient, functional and bug free.
 
 To begin contributing have a look at the open issues labeled with
-difficulty/newcomer - `newcomer issues <https://github.com/coala-analyzer/coala/issues?q=is%3Aissue+is%3Aopen+label%3Adifficulty%2Fnewcomer>`__.
+difficulty/newcomer - `newcomer issues <https://github.com/coala/coala/issues?q=is%3Aissue+is%3Aopen+label%3Adifficulty%2Fnewcomer>`__.
```
Comment (Adrianzatreanu) [inline CONTRIBUTING.rst]: i think we have a nicer link for this, coala.io/new or smt? @sils1297 
or if we dont, we should i guess

**Comment by Adrianzatreanu** [comment]
ack 9406b1ed511fcbbabee336ea70a97c5c1ab21604

**Comment by Adrianzatreanu** [comment]
ack fd99e9408539ce57459c4dd0ce3f440bd366cff8

**Comment by sils1297** [comment]
We have coala.io/new indeed

File: CONTRIBUTING.rst (hunk 3 above)
Comment (underyx) [inline CONTRIBUTING.rst]: coala.io/newcomers and coala.io/newcomer both work

Comment (underyx) [inline CONTRIBUTING.rst]: never mind, everything starting with coala.io/new works :D

**Comment by adtac** [comment]
Closing since someone else has solved this.

**Pull request closed without merging**